Content.Shared/Maps/ContentTileDefinition.cs

```diff
@@ -40,6 +40,10 @@ public sealed class ContentTileDefinition : IPrototype, IInheritingPrototype, IT
 
         [DataField("friction")] public float Friction { get; set; }
 
+        [DataField("variants")] public byte Variants { get; set; } = 1;
+
+        [DataField("placementVariants")] public List<byte> PlacementVariants { get; set; } = new() { 0 };
```

Comment:
Shouldn't variants just use the length of placementvariants?

Also should placementvariants just be an array?

Reply:
No and yes. Some tiles may have no random placement variants but multiple variant sprites. PlacementVariants is specifically "what variants can be used by the `variantize` command."

Reply:
Should have a doc comment explaining it then.